src/units/PhysicalConstexprs.hh

```diff
@@ -304,6 +312,9 @@ private:
 
   //! Electron volt value in unit system
   static constexpr double d_electronVolt = d_electronVolt_SI * Energy;
+
+  //! Atomic mass unit value in unit system
+  static constexpr double d_atomicMassUnit = d_atomicMassUnit_SI * Mass;
```

Comment:
I hadn't look at this closely before, but it appears that the use of `constexpr` with `static` prevents us from needing to declare`static` variables with `DLL_PUBLIC_units` decoration.  Nice!

Reply:
I developed PhysicalConstexprs in accordance with all the highest principles of Art